scripts/i18n/update-source.js

```diff
@@ -0,0 +1,77 @@
+const log4js = require(`log4js`)
+const shell = require(`shelljs`)
+const path = require(`path`)
+
+let logger = log4js.getLogger(`update-source`)
+logger.level = `info`
+
+const protocol = `https://`
+const host = `github.com`
+const cacheDir = `.cache`
+const owner = `gatsbyjs`
+// Repo to be used as basis for translations
+const sourceRepo = `gatsby-i18n-source`
+
+const sourceRepoUrl = `${protocol}${process.env.GITHUB_API_TOKEN}@${host}/${owner}/${sourceRepo}.git`
+
+const gatsbyMonorepoPath = path.join(__dirname, `..`, `..`)
+
+const pathsToCopy = [
+  `docs/docs`,
+  `docs/tutorial`,
+  `docs/contributing`,
+  `docs/accessibility-statement.md`,
+]
+
+function cloneOrUpdateRepo(repoName, repoUrl) {
+  if (shell.ls(repoName).code !== 0) {
+    logger.debug(`Cloning ${repoName}`)
+    const { code } = shell.exec(
+      `git clone --quiet --depth 1 ${repoUrl} > /dev/null`
+    )
+    // If cloning fails for whatever reason, we need to exit immediately
+    // or we might accidentally push to the monorepo
+    if (code !== 0) {
+      process.exit(1)
+    }
```

Comment:
I think we need to `shell.cd` into cloned repo here? We do `cd` into it in the other code path (when doing `git pull`), and seems like code after wards expect `shell` to be inside repo directory?
```suggestion
    }
    shell.cd(repoName)
```